client/dashboard/app/shopping-cart/index.tsx

```diff
@@ -0,0 +1,7 @@
+import { createShoppingCartManagerClient } from '@automattic/shopping-cart'; // eslint-disable-line no-restricted-imports
```

Comment:
Should we just allow it in eslintrc? 🤔

Reply:
I'm not totally sure. I think the lint rule is there so that we can be explicit about what existing Calypso code is being imported in the new dashboard, so that we can evaluate and refactor. But I just realized that the help center imports are already per-file, so maybe it's not a strong argument.